Routes/lobbies.route.js

```diff
@@ -0,0 +1,40 @@
+const express = require("express");
+const route = express.Router();
+
+route.get('/lobby/:id', (req, res, next) => {
+    // get lobby by id
+})
+
+route.get('/categories', (req, res, next) => {
+    // get all possible categories
+})
+
+route.post('/', (req, res, next) => {
+    //add a lobby
+})
+
+route.put('/:id', (req, res, next) => {
+    // edit a lobby
+})
+
+route.delete('/:id', (req, res, next) => {
+    //delete a lobby
+})
+
+route.put('/join/:id', (req, res, next) => {
```

Comment:
Why not POST? its not editing the lobby, its adding to the list of users who are registerin

Reply:
Ah, I see what you mean. Not sure there are wrong answers here but my thought process is basically that we are changing a single object in a collection, not adding a new one, and so it's PUT and not POST. What do you think?

Reply:
I guess it could go either way. Lets keep it as PUT for now